Microsoft.Alm.Git/GitInstallation.cs

```diff
@@ -64,6 +66,27 @@ internal GitInstallation(string path, KnownGitDistribution version)
             Debug.Assert(CommonLibexecPaths.ContainsKey(version), "The `version` parameter not found in `CommonLibExecPaths`.");
             Debug.Assert(CommonShPaths.ContainsKey(version), "The `version` parameter not found in `CommonShPaths`.");
 
+            path = path.TrimEnd('\\');
+
+            // Make sure the GitExeName isn't included as a part of the path.
+            if (path.ToLower().EndsWith(AllVersionGitPath))
```

Comment:
Sorry about that. Checked in the code I used to test. Shouldn't have. Fixed now. :)